src/pages/terms.tsx

```diff
@@ -0,0 +1,16 @@
+const TermsOfUse = () => {
```

Comment:
It is better to follow the team naming convention, the `TermsOfUse` is not representing a meaningful name for a page component. You can see the other page as example.